helm/fmtok8s-conference-chart/templates/gateway-dashboard.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: gateway
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
grafana-operator uses by default the label `app.kubernetes.io/instance` to match `GrafanaDashboard` resource.

To change it we need to add  a new label for grafana-operator (https://github.com/bitnami/charts/blob/master/bitnami/grafana-operator/README.md#grafana-parameters)